code/AutoCompleteField.php

```diff
@@ -481,19 +488,16 @@ public function Suggest(SS_HTTPRequest $request)
             return;
         }
 
-        // Find field to search within
+        // Find fields to search within
         $sourceFields = $this->getSourceFields();
 
         // input
         $q = $request->getVar('term');
         $limit = $this->getLimit();
 
         $filters = array();
-
-        foreach (preg_split('/[\s,]+/', $q) as $keyword) {
-            foreach ($sourceFields as $sourceField) {
-                $filters["{$sourceField}:PartialMatch"] = $keyword;
-            }
+        foreach ($sourceFields as $sourceField) {
+            $filters["{$sourceField}:PartialMatch"] = $q;
```

Comment:
Can you describe this change a bit? So the fix is "multi word searches must appear as type" rather than "multi word searches simply match all words anywhere"?

E.g. before `city new york` could be found by `new york` or `new york city`, but now only by the former?